examples/javascript/button.js

```diff
@@ -26,7 +26,7 @@
 var groveSensor = require('jsupm_grove');
```

Comment:
Hey, that might work (will try once I get my hands on the Edison again and update here) but requiring a separate node module for every type of sensor (especially when they're all part of the same Grove kit) seems redundant to me. Let me know what you think! :)